shared/src/business/entities/contacts/OtherPetitionerContact.test.js

```diff
@@ -1,46 +0,0 @@
-const { getOtherPetitionerContact } = require('./OtherPetitionerContact');
```

Comment:
Why'd this get deleted?

Reply:
They were no longer being used. (though they have since been re-added!)